docs/dotcom-v2-migration.md

```diff
@@ -47,8 +47,9 @@ For Carbon v11 migration guidance, see their
 | footer                    | No API changes.                                                                                                                                |
 | hr                        | View changes [here](#horizontal-rule)                                                                                                          |
 | image                     | View changes [here](#image)                                                                                                                    |
-| image-with-caption        | This component is deprecated in v1 and has been removed in v2 in favor of the image component                                                  |
+| image-with-caption        | Replaced by `image`                                                                                                                            |
```

Comment:
just curious if there is a reason for the change above? Trying to keep the docs consistent